requirements/common_constraints.txt

```diff
@@ -0,0 +1,25 @@
+# A central location for most common version constraints
```

Comment:
@Jawayria should this file be a copy or should it just be linking to the central constraints file?

Reply:
We mostly do this when we have to override some constraint but it isn't needed here so updated the PR